lightning/src/chain/channelmonitor.rs

```diff
@@ -159,176 +160,19 @@ pub enum MonitorEvent {
 	CommitmentTxBroadcasted(OutPoint),
```

Comment:
@TheBlueMatt don't you think we should have also `CounterpartyTxBroadcasted/RevokedTxBroadcasted` to reconciliate offchain state with our on-chain view as soon as we can. For revoked, we should close all our channels with a cheating peer. 

I wonder if we shouldn't just lockdown monitor at any spend of the funding transaction. I'll open an issue.